packages/components/pie-button/src/index.ts

```diff
@@ -10,7 +11,7 @@ import { BUTTON_SIZE, BUTTON_TYPE, BUTTON_VARIANT } from './defs';
 export { BUTTON_SIZE, BUTTON_TYPE, BUTTON_VARIANT };
 
 @customElement('pie-button')
-export class PieButton extends LitElement {
+export class PieButton extends RtlMixin(LitElement) {
```

Comment:
More a general q – should we expect that all components will wrap this, or only when we actually want to use it?

Mainly the q is – should we remove the mixin from pie-button as we don't need it (now that you've shown how it's used)